server/handlers.go

```diff
@@ -19,15 +19,15 @@ var ResourcesPath = filepath.Join(os.Getenv("SNAP"), "static")
 
 // PageData dynamic data to fulfill the template
 type PageData struct {
-	Ssids []netman.SSID
+	Networks []netman.SSID
```

Comment:
Quibble, but can we keep the name Ssids (not Networks)? Its more *wifi* specific.